### ISSUE TITLE
cardIndex noting working

### ISSUE THREAD

**Issue by AlmogRnD** [alexbrillant/react-native-deck-swiper#43]
I'm setting my cardIndex to 5 but it's also starting with the first card?

**Comment by KhalilKhalaf** [comment]
@AlmogRnD  Can you show full code?

**Comment by AlmogRnD** [comment]
Here is the full code 

```
render() {

        // Get connection status from state
        const connection = store.getState().connectionStatus.connection;

        // Get the selected project from state
        const selectedPhotos = store.getState().selectedTeamPhotos.photos;
        const selectedPhoto = store.getState().selectedTeamPhotos.selectedPhoto;
        const photoDescription = selectedPhoto.description;

        // Checks is selected photo is a video
        const isVideo = store.getState().selectedTeamPhotos.selectedPhoto.isVideo;

        // Gets the selected image index
        const imageIndex = _.findIndex(selectedPhotos, function(image) { return image.id == selectedPhoto.id })
        console.log(imageIndex);


        //TODO: might be better to make a separate component
        // For photo categories
        const photoCategories = selectedPhoto.categories;
        const isEmptyPhotoCategories = _.isEmpty(photoCategories);
        let categoriesOutput;
        if(!isEmptyPhotoCategories){
            categoriesOutput = Object.keys(photoCategories).map(function (key){
                return (
                    <Text  key={key}>
                        {photoCategories[key].name}
                    </Text>
                )
            })
        }

        return (
            <View style={styles.container}>
                {this.state.indeterminate &&
                <Progress.Circle
                    style={styles.progressContainer}
                    progress={this.state.progress}
                    indeterminate={this.state.indeterminate}
                />
                }

                {!connection &&
                <Text>Offline can't load image</Text>
                }

                {!isVideo &&
                <Swiper
                    ref={swiper => {this.swiper = swiper;}}
                    cards={selectedPhotos}
                    cardIndex={imageIndex}
                    renderCard={this._renderCard}
                    onSwipedLeft={this._swipeLeft}
                    showSecondCard={false}
                    backgroundColor={'#FFF'}>
                </Swiper>
                }

                {!isEmptyPhotoCategories ? <Text>{categoriesOutput}</Text>: <Text>No Categories</Text>}
                {photoDescription ? <Text>{photoDescription}</Text>: <Text>No Description</Text>}
            </View>
        )
    }
```

**Comment by AlmogRnD** [comment]
@alexbrillant any plans on fixing this in the next day or two, I have a developer milestone I need to meet and want to know if I should wait or look for another solution/package?

**Comment by AlmogRnD** [comment]
Any news regrading this?

**Comment by alexbrillant** [comment]
fixed in 1.3.3
